website/docs/node-operators/webnode/local-webnode-docker.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: Local WebNode (Docker)
+description: How to deploy and launch a webnode using Docker images
+---
+
+# Deploying and launching a webnode using Docker
+
+If you intend to just run the webnode instead of actively developing it, it is
+substantially easier to get set up using prebuilt Docker images. These images
+are automatically built on every versioned release.
+
+## Steps
+
+### 1. Generate a node key
+
+:::info This step should be redundant in a future version.
+
+:::
+
+The current version of the webnode requires you to supply a node key, even if
+you don't plan to produce blocks, or you supply your own key archive. This can
+be generated once and reused across launches of the webnode container.
+
+```sh
+# Using the latest version of the Rust node, generate a keypair.
+# The --web-node-secrets flag formats the generated keypair into JSON the webnode can use.
+docker run --rm o1labs/mina-rust:latest misc mina-key-pair --web-node-secrets > $HOME/web-node-key-pair.json
```

Comment:
Follow-up: unify the formats for keys.
We have now two formats: one for the web-node, and one for the block producer.
```
Usage: mina misc mina-key-pair [OPTIONS]

Options:
      --network <NETWORK>        Select the network (devnet or mainnet) [env: MINA_NETWORK=] [default: devnet] [possible values: devnet, mainnet]
  -s, --secret-key <SECRET_KEY>  [env: MINA_SEC_KEY=]
      --web-node-secrets         Format as a web-node-secrets.json
  -h, --help                     Print help
  ```
  
We should only have one (encrypted) format, and use it across all the nodes (OCaml, Rust, Web).